code/modules/projectiles/projectile/special.dm

```diff
@@ -215,7 +216,6 @@
 			name = "full strength [name]"
 			damage *= 4
 			range *= 2
-	..()
```

Comment:
This was running code before parent New() that would now happen after New().

Does this matter in this case?